From 25132cbb589ac292d74003b2f3acbe598bcd7589 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Thu, 10 Nov 2022 20:07:12 +0100
Subject: [PATCH 360/389] media: i2c: ov8858: Use default subdev name

This results in names like "ov8858 0000:0001" which are more in line
with mainline conventions. This increases compatibility with future
mainline driver.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 drivers/media/i2c/ov8858.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/media/i2c/ov8858.c b/drivers/media/i2c/ov8858.c
index 5a66cb65a48c..55d2daa82e57 100644
--- a/drivers/media/i2c/ov8858.c
+++ b/drivers/media/i2c/ov8858.c
@@ -2933,9 +2933,6 @@ static int ov8858_probe(struct i2c_client *client,
 	else
 		facing[0] = 'f';
 
-	snprintf(sd->name, sizeof(sd->name), "m%02d_%s_%s %s",
-		 ov8858->module_index, facing,
-		 OV8858_NAME, dev_name(sd->dev));
 	ret = v4l2_async_register_subdev_sensor(sd);
 	if (ret) {
 		dev_err(dev, "v4l2 async register subdev failed\n");
-- 
2.35.3

